Tests/Tests/Event Logging/EventLoggingTests.swift

```diff
@@ -12,7 +12,8 @@ class EventLoggingTests: XCTestCase {
     private let domain = "event-logging-tests.example"
     lazy var url = URL(string: "http://\(domain)")!
 
-    func testThatOnlyOneFileIsUploadedSimultaneously() {
+    func testThatOnlyOneFileIsUploadedSimultaneously() throws {
+        throw XCTSkip("This test seems to be flaky")
```

Comment:
I added this here because [the failures](https://buildkite.com/automattic/tracks-ios/builds/423#018bd479-0ae7-4556-a14e-1f64102ef59e) don't seem to be related to any code changes. 

@jkmassel recently added an additional XCTSkip to another test in this file. 

I created https://github.com/Automattic/Automattic-Tracks-iOS/issues/270 so that we can track this and investigate what's causing the flakiness.